### PR TITLE
UI: `Tooltip`

### LINKED ISSUE #78: Create UI components with Reach UI or Radix UI
- [x] [Text](https://minimals.cc/components/typography) #81 
- [x] [Snackbar](https://minimals.cc/components/snackbar) #98 
- [x] [Tab](https://minimals.cc/components/tabs) #118
- [x] [Menu](https://minimals.cc/components/menu)
     * #119 
     * #120
- [ ] [Tooltips](https://minimals.cc/components/tooltip) #129 
- [ ] [Chip](https://minimals.cc/components/chip)

### PR DESCRIPTION
**Pull request by vitokhangnguyen**
Resolve #78

### CODE REVIEW

File: src/ui/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  /**
+   * *A single {@link JSX.Element}* as the handle to control the tooltip popover to be rendered where {@link Tooltip} is used.
+   *
+   * **Note:** A valid trigger here must be an HTML native element or an element that implement the {@link React.forwardRef} API.
+   */
+  trigger: ReactNode;
+  /**
+   * Theme color to be applied on the tooltip popover
+   */
+  theme?: 'dark' | 'light' | 'primary' | 'secondary';
```
Comment (Andrewnt219) [inline src/ui/Tooltip/Tooltip.tsx:16]: Appreciate the customizable, but I don't think adding `theme` to every component is necessary.

File: src/ui/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      <RadixTooltip.Trigger asChild>{trigger}</RadixTooltip.Trigger>
+      <AnimatePresence>
+        {/* Required to aniamte exit */}
+        {showTooltip && (
+          <RadixTooltip.Content
+            side="top"
+            sideOffset={4}
+            {...menuPopupProps}
+            asChild
+            forceMount
```
Comment (Andrewnt219) [inline src/ui/Tooltip/Tooltip.tsx:55]: what is `forceMount`?

Comment (vitokhangnguyen) [reply src/ui/Tooltip/Tooltip.tsx:55]: 🤷 

According to Radix UI's docs:

```
forceMount?: true | undefined: Used to force mounting when more control is needed. Useful when controlling animation with React animation libraries.
```

Exit aniamtion doesn't work without this

File: src/ui/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (vitokhangnguyen) [reply src/ui/Tooltip/Tooltip.tsx:16]: Removed theme prop

**Review by Andrewnt219 — APPROVED** [review]

**Pull request merged**